### PR TITLE
Add aws-full command and refactor to remove workspaces

### PR DESCRIPTION
**Pull request by paurosello**
Execute pl002, pl003, pl004 and pl005 to execute all test.

### CODE REVIEW

**Review by calvix — APPROVED** [review]

File: tekton/tasks/aws-pl002.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: aws-full
```
Comment (MarcelMue) [inline tekton/tasks/aws-pl002.yaml:4]: All of these have the same name - this won't work I believe.

File: tekton/tasks/aws-pl002.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  namespace: test-workloads
+spec:
+  volumes:
+    - name: kubeconfig
+      secret:
+        secretName: standup-kubeconfig
+  workspaces:
+    - name: cluster
+      description: Run the aws conformance test plan to test upgrades.
+  steps:
```
Comment (MarcelMue) [inline tekton/tasks/aws-pl002.yaml:14]: A lot of these plan tasks look extremely similar. Can we maybe have one or two generic tasks with the plan name as input?

File: tekton/tasks/aws-pl002.yaml (hunk 1 above)
Comment (paurosello) [reply tekton/tasks/aws-pl002.yaml]: yeah, already changed

File: tekton/tasks/aws-pl002.yaml (hunk 2 above)
Comment (paurosello) [reply tekton/tasks/aws-pl002.yaml:14]: hmm that is a good idea, will try to refactor this

**Review by MarcelMue — APPROVED** [review]
This is now looking v good to me!

Did you test this already?

**Comment by paurosello** [comment]
I am currently testing, it seems there are some problems when executing multiple tasks with the same "workspace" like https://tekton.giantswarm.io/#/namespaces/test-workloads/pipelineruns/15c6597f-fb42-11eb-959d-fa3ca8521a07?pipelineTask=test-aws-pl002&step=run-tests

I will try to make this sequential and see if it works.

**Comment by paurosello** [comment]
@MarcelMue do you have an idea why do I get this on the new pipeline I created:

https://tekton.giantswarm.io/#/namespaces/test-workloads/pipelineruns/1f6df920-fb54-11eb-bb04-b613c42b326b?pipelineTask=test-aws-pl002&step=run-tests&view=logs

```
panic: {"kind":"unknown","annotation":"unknown flag: --output","stack":[{"file":"/root/project/main.go","line":78}]}

goroutine 1 [running]:
main.main()
	/root/project/main.go:28 +0x2cf
```

File: tekton/tasks/aws-cnfm.yaml [hunk 3]
```diff
@@ -10,18 +10,26 @@ spec:
...
         
         export AWSCNFM_CONTROLPLANE_KUBECONFIG=/etc/controlplane/aws
-        export AWSCNFM_CREATE_RELEASEVERSION=$(cat $(workspaces.cluster.path)/release-id)
-        
-        awscnfm plan pl001 \
+        export AWSCNFM_CREATE_RELEASEVERSION=v14.2.2 
+        export AWSCNFM_UPDATE_RELEASEVERSION=$(cat $(workspaces.cluster.path)/release-id)
+
+        awscnfm plan $CNFM_PLAN \
         --output $(workspaces.cluster.path)
```
Comment (MarcelMue) [inline tekton/tasks/aws-cnfm.yaml:35]: Does every plan have `--output`?

Comment (paurosello) [reply tekton/tasks/aws-cnfm.yaml:35]: I think so, my feeling is that the cluster var was not being filled for some reason but will check locally

Comment (paurosello) [reply tekton/tasks/aws-cnfm.yaml]: I think that made it... strange. Wonder if I broke something here but looks better

Comment (MarcelMue) [reply tekton/tasks/aws-cnfm.yaml]: Huh not sure to be honest. I don't know much about awscnfm

**Comment by T-Kukawka** [comment]
towards: https://github.com/giantswarm/roadmap/issues/376

**Pull request closed without merging**